tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1947,4 +1956,102 @@ TEST_F(EnergyPlusFixture, MultiSpeedCoolingCoil_SEERValueTest)
+
+    // set AHRI test load ratios
+    Real64 constexpr LoadRatio100 = 1.0;
+    Real64 constexpr LoadRatio75 = 0.75;
+    Real64 constexpr LoadRatio50 = 0.50;
+    Real64 constexpr LoadRatio25 = 0.25;
+
+    // Test 1: WaterCooled Condenser Entering Water Temperatures
+    DataPlant::CondenserType CondenserType = DataPlant::CondenserType::WaterCooled;
+    StandardRatings::AhriChillerStd Ahri_Chiller_Std = StandardRatings::AhriChillerStd::AHRI550_590;
+    Real64 result_EWT100LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio100);
+    Real64 result_EWT75LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio75);
+    Real64 result_EWT50LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio50);
+    Real64 result_EWT25LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio25);
+    // checking expected water temperatures
+    EXPECT_NEAR(29.44, result_EWT100LoadRatio, 0.01);
+    EXPECT_NEAR(23.89, result_EWT75LoadRatio, 0.01);
+    EXPECT_NEAR(18.33, result_EWT50LoadRatio, 0.01);
+    EXPECT_NEAR(18.33, result_EWT25LoadRatio, 0.01);
+
+    // Test 2: AirCooled Condenser Entering Air Dry-Bulb Temperatures
+    CondenserType = DataPlant::CondenserType::AirCooled;
+    Real64 result_EDBT100LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio100);
+    Real64 result_EDBT75LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio75);
+    Real64 result_EDBT50LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio50);
+    Real64 result_EDBT25LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio25);
+    // checking expected dry-bulb temperatures
+    EXPECT_NEAR(35.0, result_EDBT100LoadRatio, 0.01);
+    EXPECT_NEAR(26.67, result_EDBT75LoadRatio, 0.01);
+    EXPECT_NEAR(18.33, result_EDBT50LoadRatio, 0.01);
+    EXPECT_NEAR(12.78, result_EDBT25LoadRatio, 0.01);
+
+    // Test 3: EvapCooled Condenser Entering Air Wet-Bulb Temperatures
+    CondenserType = DataPlant::CondenserType::EvapCooled;
+    Real64 result_EWBT100LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio100);
+    Real64 result_EWBT75LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio75);
+    Real64 result_EWBT50LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio50);
+    Real64 result_EWBT25LoadRatio = CondenserEnteringFluidTemperature(CondenserType, Ahri_Chiller_Std, LoadRatio25);
+    // checking expected wet-bulb temperatures
+    EXPECT_NEAR(23.89, result_EWBT100LoadRatio, 0.01);
+    EXPECT_NEAR(20.42, result_EWBT75LoadRatio, 0.01);
+    EXPECT_NEAR(16.94, result_EWBT50LoadRatio, 0.01);
+    EXPECT_NEAR(13.47, result_EWBT25LoadRatio, 0.01);
+}
```

Comment:
Unit test for AHRI Std 550/590 (IP) condenser entering fluid temperatures for three condenser types and load ratios.